### DISCUSSION TITLE
Export data sources to provisioning datasource.yml

### DISCUSSION THREAD

**Discussion by kevinlinxc** [grafana/grafana#68844] [Data Sources]
The context is that I am running Grafana locally, either with the server executable or with Docker.

Currently, I think you have to manually trial and error your `provisioning/datasources/datasource.yml` file until the datasource is correctly provisioned. However, it is much easier to iterate on the Grafana UI when adding a datasource, so why not make the datasources on the UI exportable to a yml file? Same goes for dashboards.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82851. Feel free to continue the discussion around this there. Thank you!
